src/engine/objects/robot/sensors/SimColorSensor.ts

```diff
@@ -79,8 +79,8 @@ export class SimColorSensor extends SimComplexSensor {
       density: 1,
       isSensor: true,
       userData,
-      filterCategoryBits: EntityCategory.SENSORS,
-      filterMaskBits: EntityMask.SENSORS,
```

Comment:
I've read more about filter collision and I realize this is wrong.
I think we don't need filter category for sensor, but the important fix is to add ZONES in `EntityMask`

https://github.com/FRUK-Simulator/SimulatorCore/blob/bada785bfeef72e37b09cf3feceb5a029548df0a/src/engine/objects/robot/RobotCollisionConstants.ts#L33